nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -176,6 +176,9 @@ def _create_statistics_aggregator(self,
             from nncf.experimental.openvino_native.statistics.aggregator import \
                 OVStatisticsAggregator
             return OVStatisticsAggregator(dataset)
+        if backend == BackendType.TORCH:
+            from nncf.torch.statistics.aggregator import PTStatisticsAggregator
```

Comment:
Would the PyTorch as a backend for PTQ be presented in common, not as `experimental` at first?